### ISSUE TITLE
Uint8Array is not supported in React Native

### ISSUE THREAD

**Issue by JadeTao** [Game-X-Coin/eosjs-rn#11]
**Version of EOSJS**
beta3

**Describe the bug**
`src/eosjs-numeric.ts`
Uint8Array is not supported in React Native

**To Reproduce**
just use.

`Uint8Array`  should be polyfilled.

**Comment by JadeTao** [comment]
i am confusing now whether react native supports `Uint8Array`...😢
at least the latest version does not support

**Comment by JadeTao** [comment]
`core-js` helps me out.
just `import 'core-js'` in the 1st line of `index.js`.

**Comment by JadeTao** [comment]
if maintainers have nothing to say, just close it.

**Comment by ehnawh** [comment]
When I tried it, Uint8Array was running without any problems.

**Comment by JadeTao** [comment]
> When I tried it, Uint8Array was running without any problems.

int the latest version?

**Comment by ehnawh** [comment]
> > When I tried it, Uint8Array was running without any problems.
> 
> int the latest version?

Yes. i used beta3 version. It worked well in beta2, but does beta3 have an error?
How did you use JsSignatureProvider?
A : import JsSignatureProvider from 'eosjs-rn/dist/eosjs-jssig'; 
or
B :  import { JsSignatureProvider } from 'eosjs-rn'

**Comment by JadeTao** [comment]
both, all failed.

and sry I mean the latest react-native.

**Comment by ehnawh** [comment]
i 

> both, failed.
> 
> and sry I mean the latest react-native.

I tried it in react-native 0.57 version, but it worked. Can I see the code that failed the error?

**Comment by JadeTao** [comment]
https://github.com/Game-X-Coin/eosjs-rn/issues/10#issuecomment-450474065

here. thanks for patience.

**Comment by ehnawh** [comment]
Please follow the procedure below.
1. remove eosjs module
2. install eosjs-rn module
3. remove node_modules folder and remove *.lock file
4. install module. yarn or npm install
5. run app

**Comment by JadeTao** [comment]
> I have seen it run successfully.
> Please follow the procedure below.
> 
> 1. remove eosjs module. 'npm uninstall eosjs' or 'yarn remove eosjs'
> 2. install eosjs-rn module. 'npm install' eosjs-rn' or 'yarn add eosjs-rn'
> 3. change module eosjs -> eosjs-rn to App.js
> 4. run app.
> 
> If you have followed the steps above, but you can not do it, do the following:
> 
> 1. remove node_modules folder and remove *.lock file
> 2. install module. yarn or npm install

sry not described this issue properly.

i did the same steps.
and and the mistake can only be reproduced in Android (without remote debugger)

**Comment by ehnawh** [comment]
> > I have seen it run successfully.
> > Please follow the procedure below.
> > 
> > 1. remove eosjs module. 'npm uninstall eosjs' or 'yarn remove eosjs'
> > 2. install eosjs-rn module. 'npm install' eosjs-rn' or 'yarn add eosjs-rn'
> > 3. change module eosjs -> eosjs-rn to App.js
> > 4. run app.
> > 
> > If you have followed the steps above, but you can not do it, do the following:
> > 
> > 1. remove node_modules folder and remove *.lock file
> > 2. install module. yarn or npm install
> 
> sry not described this issue properly.
> 
> i did the same steps.
> and and the mistake can only be reproduced in Android (without remote debugger)

**Comment by ehnawh** [comment]
https://github.com/Game-X-Coin/eosjs-rn/issues/11#event-2059099945

**Comment by JadeTao** [comment]
yes. and `core-js` can also help.
just import 'core-js' in the 1st line of index.js.

**Comment by ajitsen** [comment]
you could also just do, to avoid global namespace pollution

import Int8Array from 'core-js/fn/typed/int8-array';
